src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstier/domain/TierCalculation.kt

```diff
@@ -145,29 +145,46 @@ class TierCalculation {
 
     private fun getComplexityPoints(
       complexityFactors: List<ComplexityFactor>,
-      assessmentComplexityFactors: Map<AssessmentComplexityFactor, String?>
+      assessmentComplexityFactors: Map<AssessmentComplexityFactor, String?>,
+      isFemale: Boolean
     ): Int {
 
       return if (complexityFactors.any() || assessmentComplexityFactors.any()) {
         criteria.add(TierMatchCriteria.INCLUDED_COMPLEXITY_FACTORS)
-        (complexityFactors.distinct().count().plus(getAssessmentComplexityPoints(assessmentComplexityFactors))) * 2
+        val multiplicationFactor = 2
+        val genericScore = complexityFactors.distinct().count() * multiplicationFactor
+
+        when (isFemale) {
+            true -> {
+                genericScore.plus(getAssessmentComplexityPoints(assessmentComplexityFactors) * 2)
+            }
+            else -> {
+              when {
+                complexityFactors.contains(ComplexityFactor.IOM_NOMINAL) -> {
+                  genericScore - multiplicationFactor
```

Comment:
Again, knowing nothing of Kotlin so please ignore if irrelevant. If there is a .plus function (159) should this use .minus?

Also a variable called multiplicationFactor could perhaps have a better name if it's being subtracted rather than multiplied (unless it is a domain specific name of course)

Reply:
No you're right, good catch, I should use .minus() to be consistent. I'll also rename multiplicationFactior and put a comment, I should also use it on line 159 where the '2' is.